app/views/honesty_declaration.scala.html

```diff
@@ -0,0 +1,57 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.play.views.html.helpers
+@import config.AppConfig
+@import models.auth.User
+@import views.html.templates.inputs.checkboxHelper
+
+@(periodKey: String, form: Form[Boolean])(implicit request: Request[_], messages: Messages, appConfig: AppConfig, user: User[_])
+
+@statement = {
+    @if(user.isAgent) {
+        @messages("honesty_declaration.agentStatement")
+    } else {
+        @messages("honesty_declaration.statement")
+    }
+}
+
+@*TODO - update pageTitle to include error prefix if form contains error*@
+@main_template(messages("honesty_declaration.title"), appConfig = appConfig, bodyClasses = None, user = Some(user)) {
```

Comment:
Can you remove the `bodyClasses = None` from this constructor. It's set to `None` by default anyway in `main_template`